src/index.html

```diff
@@ -17,11 +17,388 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/index.scss"
```

Comment:
You are linking a SCSS file in your HTML. Browsers cannot interpret SCSS, they only understand CSS. You need to compile your SCSS into CSS and then link the CSS file in your HTML. You can use tools such as Sass or Gulp to compile your SCSS into CSS.